### PR TITLE
Implemented a very simple language switcher;

### PR DESCRIPTION
**Pull request by cyanthvibes**
Default language is Dutch, and user preference is stored in localStorage.

Large amount of file changes due to optimising imports and forcing ESlint rules across the whole project.

### CODE REVIEW

File: src/app/resident/resident.module.ts [hunk 1]
```diff
@@ -18,6 +21,9 @@ import { MatButtonModule } from '@angular/material/button';
     MatListModule,
     RouterModule,
     MatButtonModule,
+    TranslateModule,
+    ManagerModule,
```
Comment (WouterRBerkhout) [inline src/app/resident/resident.module.ts:25]: Why are you importing the manager module into the resident module? Seems counter intuitive

**Review by WouterRBerkhout — CHANGES_REQUESTED** [review]
Please keep resident and manager modules seperate

File: src/app/resident/resident.module.ts (hunk 1 above)
Comment (WouterRBerkhout) [reply src/app/resident/resident.module.ts:25]: Will merge for demo purposes, but will change later

**Review by WouterRBerkhout — APPROVED** [review]

File: src/app/resident/resident.module.ts (hunk 1 above)
Comment (cyanthvibes) [reply src/app/resident/resident.module.ts:25]: Might've been a result of my own testing and therefore clowncode. Will alter this in a future commit when styling has been properly done for the language switcher ;)

**Pull request merged**